client/dashboard/task-list/tasks/payments/index.js

```diff
@@ -443,6 +462,24 @@ class Payments extends Component {
 						>
 							{ __( 'Proceed', 'woocommerce-admin' ) }
 						</Button>
+						{ industryIsCBD && (
+							<Tooltip
```

Comment:
https://github.com/woocommerce/woocommerce-admin/pull/3782 updates the flow to set up individual gateways, so we no longer have a single "Proceed" button.

Should this tooltip be moved next to the new "Set up" buttons? /cc @jameskoster @pmcpinto

Reply:
Per slack discussion in p1583403783011700-slack-wc-onboarding it looks like we won't use the tooltip, but instead use the "recommended ribbon."  See https://github.com/woocommerce/woocommerce-admin/blob/a7dc0d16d2b7a9a2bdc505b7b1a24bbb81ad0553/client/dashboard/task-list/tasks/payments/index.js#L43 for implementation.

Reply:
Added this change in the commit [216018a](https://github.com/woocommerce/woocommerce-admin/pull/3745/commits/216018aca4c2b6533811e3657c305a681c64626d)